app/views/notes/index.html.erb

```diff
@@ -15,37 +15,28 @@
   <% else %>
     <h2><%= raw translation('notes.index.research_notes_ideas_and_documentation') %></h2>
   <% end %>
-  <% if params[:controller] == "notes" && (params[:action] == "popular" || params[:action] == "index" || params[:action] == "liked" || params[:action] == "recent") %>
-
-    <%= render :partial => "notes/format_toggle" %>
-    <% if params[:action] == "liked" %><a style="margin-right:4px;" class="float-right btn btn-outline-secondary btn-sm" href="/feed/liked.rss"><i class="fa fa-rss"></i></a><% end %>
+  <br/>
+  <% if params[:action] == "liked" %><a style="margin-right:9px;" class="float-right btn btn-outline-secondary btn-sm" href="/feed/liked.rss"><i class="fa fa-rss"></i></a><% end %>
 
-    <ul class="nav nav-tabs" id="tab-bar">
+    <ul class="nav nav-pills">
       <li class="nav-item">
-        <a class="nav-link <% if params[:action] == "recent" %> active <% end %>" href="/notes/recent"> <i class="fa fa-clock-o"></i><span class="hidden-sm hidden-xs"> <%= translation('notes.index.recent') %></span></a>
+        <a class="nav-link <% if params[:controller] == "notes" %> active <% end %>" href="/notes/" target="_blank"> <i class="fa fa-file"></i><span class="hidden-sm hidden-xs"> <%= translation('notes.index.research_notes') %></span></a>
```

Comment:
Should these be target blank? Perhaps we could have them be regular links? but i'm very open to ideas if you'd like to share your reasoning! Thanks!

Reply:
Good catch, the `research_notes` link should not be target blank.. but I think for the rest of the links it makes sense for them to be since once you click on either `question, wikis, or comments` there's no way of going back to the notes section from those pages... unless you go back to the dashboard then click the 'all posts' link which is an extra step for the user in loading and reloading the pages... if this is not a problem, I can remove the target blank from the rest of the links too.

Reply:
I think we could go with normal links for all, but i see your point too. Maybe we can brainstorm a bit on the UI design for this in follow-up (like, could we put a compact way to navigate back on the other pages? not sure.); i think your solution is good too, but we needn't let this hold us up. Thanks for thinking carefully and being flexible here!

Reply:
Alright, I'll remove the target blank and we can follow it up more in the future..